cms/jinja2/templates/pages/release_index.html

```diff
@@ -1 +1,8 @@
 {% extends "templates/base_page.html" %}
+{% block header_area %}
```

Comment:
Should this display the breadcrumb? I'm not seeing it on the release calendar page.

Reply:
The release calendar index page should not have a breadcrumb